packages/loader/container-loader/src/test/containerContext.spec.ts

```diff
@@ -153,20 +168,28 @@ describe("ContainerContext Tests", () => {
         mockCodeLoader.verify();
     });
 
+    // open an issue to add string typing
     it("Should load code with details", async () => {
         // Arrange
         const proposedCodeDetails = codeDetailsForVersion("2.0.0");
         const moduleCodeDetails = codeDetailsForVersion("3.0.0");
+        const load = async (): Promise<IFluidModuleWithDetails> => {
+            return {
+                module: { fluidExport: { } },
+                details: proposedCodeDetails,
+            };
+        };
 
         const codeDetailsLoader = {
-            load: async () => {},
+            load,
             get IFluidCodeDetailsComparer() {
                 return this;
             },
             satisfies: async (
                 candidate: IFluidCodeDetails,
                 constraint: IFluidCodeDetails,
-            ) => {},
+            ) => { return true; },
+            compare: async () => { return 0; },
```

Comment:
What is the purpose of this change?

Reply:
IFluidCodeDetailsComparer has two required functions: satisfies and compare. After typing ICodeDetailsLoader, compare was missing.

Reply:
Not sure I fully understand, but it sounds like you are clear on the purpose and validity of this test.